config/initializers/ac_mediainfo.rb

```diff
@@ -1,6 +0,0 @@
-require 'mediainfo'
-
-Mediainfo.path = Settings.mediainfo.path if Settings&.mediainfo&.path
-
-# Set up for active_encode, need to happen before active_encode initializer
-ENV["MEDIAINFO_PATH"] ||= Mediainfo.path 
```

Comment:
Is this not needed by ActiveEncode anymore?

Reply:
Oh, good call. That file is probably still necessary.